setup.py

```diff
@@ -69,7 +69,7 @@ def build_extensions(self):
     Extension('gmpy2.gmpy2',
               sources=sources,
               include_dirs=['./src'],
-              libraries=['mpc','mpfr','gmp'],
+              libraries=['mpc','mpfr','gmp','m'],
```

Comment:
Can you do this only for unix platforms?

Reply:
Done.